hash/msx1_cart.xml

```diff
@@ -22544,7 +22541,7 @@ legacy FM implementations cannot find it.
 	<software name="lacorencf" cloneof="lacorenc">
 		<description>La Corona Encantada (Spanish, final)</description>
 		<year>2009</year>
-		<publisher>&lt;homebrew&gt;</publisher>
+		<publisher>RELEVO Videogames</publisher>
```

Comment:
It’s always “Relevo” in text on their web site.  It’s only stylised “RELƎVO” with a turned E from the Pan-Nigerian alphabet in the logo image.